docs/activity.md

```diff
@@ -25,14 +25,18 @@ public class Example {
 A command activity is triggered when a message is sent in an `IM`, `MIM` or `Chatroom`. This is the most basic interaction 
 between an end-user and the bot. Here are some command activity examples: 
 
+- the bot is mentioned followed by a [_slash_](#slash-command) command:
```

Comment:
Where is this link supposed to go ?

Reply:
Here : https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/master/docs/activity.md#slash-command